notification-service/src/main/resources/application.yml

```diff
@@ -25,19 +18,20 @@ spring:
             enable: true
             required: true
     verify:
-      host: ${VERIFY_EMAIL_HOST}
+      host: http://localhost:587
```

Comment:
Values that change depending on the environment (in this case: host, port) should be moved to be in the .env file.

Reply:
Done.